docs/start/start.md

```diff
@@ -6,11 +6,11 @@ filterKey: integration
 ---
 
 The open-source Amplify Framework provides the following products to build fullstack iOS, Android, Web, and React Native apps:
-- **Amplify [CLI](../cli)** - Configure all the services needed to power your backend through a simple command line interface.
-- **Amplify [Libraries](../lib)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
-- **Amplify UI components** - UI libraries for React, React Native, Angular, Ionic and Vue.
 
-The **Amplify [Console](https://aws.amazon.com/amplify/console/)** is an AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
+- **Amplify [CLI](~/cli/cli.md)** - Configure all the services needed to power your backend through a simple command line interface.
+- **Amplify [Libraries](~/lib/lib.md)** - Use case-centric client libraries to integrate your app code with a backend using declarative interfaces.
+- **Amplify [UI components](~/ui/ui.md)** - UI libraries for React, React Native, Angular, Ionic and Vue.
+- **Amplify [Console](https://aws.amazon.com/amplify/console/)** – An AWS service that provides a git-based workflow for continuous deployment & hosting of fullstack web apps. Cloud resources created by the Amplify CLI are also visible in the Amplify Console.
```

Comment:
This needs to be in a separate line (kinda) like what it is in prod with a <br/> @brene